src/main/java/net/neoforged/neoforge/client/textures/FluidSpriteCache.java

```diff
@@ -42,6 +42,13 @@ public static TextureAtlasSprite[] getFluidSprites(BlockAndTintGetter level, Blo
         };
     }
 
+    /**
+     * {@return a specified sprite or a missing sprite texture if sprite is not found.
```

Comment:
You forgot the closing `}`, and the final `.` is not needed (it's already added by `{@return }`).